pybv/io.py

```diff
@@ -48,7 +47,18 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
     sfreq : int | float
         The sampling frequency of the data.
     ch_names : list of strings, shape (n_channels,)
-        The name of each channel.
+        The names of the channels.
+    ref_ch_names : str | list of strings, shape (n_channels,) | None
+        The name of the channel used as a reference during the recording. If
+        references differed between channels, you may supply a list of
+        reference channel names corresponding to each channel in ``ch_names``.
+        If ``None`` (default), assume the common average reference.
```

Comment:
> Specifying `None` and writing nothing is not the same as the CAR - it just means that the information is not present.

The specs say otherwise:
<img width="1318" alt="Screen Shot 2021-06-21 at 11 13 43" src="https://user-images.githubusercontent.com/2046265/122738422-416a5080-d282-11eb-9439-4805e5749f0b.png">



But I do see where the misunderstanding comes from, I shouldn't have written "common average" but "reference-free" (although my understanding of looking at the circuits of the ActiCHamp amplifier we had in our old lab was that internally it **would** reference mean across all channels):

<img width="1021" alt="Screen Shot 2021-06-21 at 11 16 21" src="https://user-images.githubusercontent.com/2046265/122738760-9312db00-d282-11eb-89b2-330146c54110.png">


<img width="421" alt="Screen Shot 2021-06-21 at 11 15 31" src="https://user-images.githubusercontent.com/2046265/122738792-98702580-d282-11eb-8bc5-f3c1ce308638.png">


In any case, I propose to change the wording to "If None, assumes reference-free recording", WDYT?



> This is the beginning from a VHDR file I recorded using the standard BrainVision equipment, software, and workflow (Reference is on position FCz, but does not occur as a channel in the data):

This seems to be violating their own file spec 🙈😱
Unless … I believe in BV Recorder there's the option to do a reference-free recording, BUT still select a reference electrode merely for online data viz. Could that have been the case here?

Reply:
Good point you make in #74 though, maybe we should reach out to BP for clarification.

Reply:
I am pretty sure BV recorder doesn't have a "reference free" button :thinking: but yes, there seems to be some sort of conflict :-(

I am going to write to BP support about this and will put you in cc / post the history here.

Reply:
btw: history is posted in #74